tests/test_result.py

```diff
@@ -44,6 +113,18 @@ def test_raw_probability(state):
     assert target_dict == results.raw_probability(state=state)
 
 
+@pytest.mark.parametrize("state", [0, 1])
+def test_state_probability(state):
+    """Testing raw_probability method"""
+    results = generate_random_state_result(5)
+    if state == 0:
+        target_dict = {"probability": results.state_0_probability}
+    else:
+        target_dict = {"probability": 1 - results.state_0_probability}
```

Comment:
```suggestion
        target_dict = {"probability": results.probability(0)}
    else:
        target_dict = {"probability": 1 - results.probability(0)}
```

Hopefully the last issue for tests.